ui/packages/consul-ui/tests/acceptance/dc/acls/list-order.feature

```diff
@@ -1,40 +0,0 @@
-@setupApplicationTest
-@notNamespaceable
-Feature: dc / acls / list-order
-  In order to be able to find ACL tokens easier
-  As a user
-  I want to see the ACL listed alphabetically by Name
-
-  Scenario: I have 10 randomly sorted tokens
-    Given 1 datacenter model with the value "datacenter"
-    And I'm using a legacy token
-    And 10 acl model from yaml
-    ---
-      - Name: zz
-      - Name: 123
-      - Name: aa
-      - Name: 9857
-      - Name: sfgr
-      - Name: foo
-      - Name: bar
-      - Name: xft
-      - Name: z-35y
-      - Name: __acl
-    ---
-    When I visit the acls page for yaml
-    ---
-      dc: datacenter
-    ---
-    Then I see name on the acls like yaml
-    ---
-      - __acl
-      - 123
-      - 9857
-      - aa
-      - bar
-      - foo
-      - sfgr
-      - xft
-      - z-35y
-      - zz
-    ---
```

Comment:
Anything problematic to do with legacy ACLs I just deleted. It would be wasted work to make sure anything to do with legacy ACLs continued to pass/work seeing as we are removing legacy ACLs for 1.11.

I specifically only removed things that were problematic in some way and left the actual removal of legacy ACLs for a further PR.